Prevents error:
/bin/sh: 1: /data/data/com.termux/files/usr/bin/glslc: Exec format error
during cross-compilation if shaderc was built in the same termux-package-builder container
before building openxr.
glslc is optional for building openxr, since normally it builds successfully without it.

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -341,11 +341,6 @@ else()
             ${VULKAN_SDK}/*
         )
     endif()
-    find_program(
-        GLSL_COMPILER glslc
-        PATHS ${glslc_folders}
-        HINTS "${Vulkan_GLSLC_EXECUTABLE}"
-    )
 endif()
 find_program(
     GLSLANG_VALIDATOR glslangValidator
